src/macaron/console.py

```diff
@@ -0,0 +1,390 @@
+                if self.components_violates_table.row_count > 0:
+                    layout = layout + [
+                        "[bold red] Components Violates Policy[/]",
+                        self.components_violates_table,
+                    ]
+                else:
+                    layout = layout + [
+                        "[bold red] Components Violates Policy[/]   [white not italic]None[/]",
+                    ]
+                if self.components_satisfy_table.row_count > 0:
+                    layout = layout + [
+                        "",
+                        "[bold green] Components Satisfy Policy[/]",
+                        self.components_satisfy_table,
+                    ]
+                else:
+                    layout = layout + [
+                        "",
+                        "[bold green] Components Satisfy Policy[/]  [white not italic]None[/]",
+                    ]
+                layout = layout + ["", self.policy_summary_table]
+                if self.verification_summary_attestation:
+                    vsa_table = Table(show_header=False, box=None)
+                    vsa_table.add_column("Detail", justify="left")
+                    vsa_table.add_column("Value", justify="left")
+
+                    vsa_table.add_row(
+                        "[bold blue]Verification Summary Attestation[/]",
+                        self.verification_summary_attestation,
+                    )
+
+                    layout = layout + [vsa_table]
+        elif self.command == "find-source":
+            if self.find_source_table.row_count > 0:
+                layout = layout + [self.find_source_table]
+        elif self.command == "dump-defaults":
+            dump_defaults_table = Table(show_header=False, box=None)
+            dump_defaults_table.add_column("Detail", justify="left")
+            dump_defaults_table.add_column("Value", justify="left")
+            dump_defaults_table.add_row("Dump Defaults", self.dump_defaults)
+            layout = layout + [dump_defaults_table]
+        elif self.command == "gen-build-spec":
+            if self.gen_build_spec_table.row_count > 0:
+                layout = layout + [self.gen_build_spec_table]
+        if self.verbose:
+            layout = layout + ["", self.verbose_panel]
+        return Group(*layout)
+
+    def start(self, command: str) -> None:
+        """Start the live console display."""
+        self.command = command
+        if not self.live.is_started:
+            self.live.start()
+
+    def close(self) -> None:
+        """Stop the live console display."""
+        self.live.stop()
+
+
+class AccessHandler:
+    """A class to manage access to the RichConsoleHandler instance."""
+
+    def __init__(self) -> None:
+        self.rich_handler = RichConsoleHandler()
+
+    def set_handler(self, verbose: bool) -> RichConsoleHandler:
+        """Set the verbosity and create a new RichConsoleHandler instance."""
```

Comment:
Please make sure to include `Parameters`, `Returns`, and if needed `Raises` for exceptions in the docstrings. This comment applies to all the docstrings.

See our contribution guideline here: https://oracle.github.io/macaron/pages/developers_guide/style_guide.html#docstrings